tools/validators/instance_validator/tests/entity_instance_test.py

```diff
@@ -33,10 +33,11 @@ def setUp(self):
     self.universe = generate_universe.BuildUniverse()
 
   def testValidateGoodExample(self):
-    parsed = dict(instance_parser.parse_yaml(
+    parsed, _ = instance_parser.parse_yaml(
         os.path.join(_TESTCASE_PATH,
```

Comment:
`_` I prefer that you put here what you are expecting to be returned I think exception and assert it is empty in your test

Reply:
same for the ones below in your tests

Reply:
Sounds good